doc/sphinx/Makefile

```diff
@@ -35,12 +36,12 @@ doxy-code: gen_python_api_doc
 html: doxy-code
 
 pickle:
-	$(SPHINXBUILD) -b pickle $(ALLSPHINXOPTS) $(BUILDDIR)/pickle
+	$(SPHINXBUILD) -b pickle $(SOURCEDIR) $(BUILDDIR)/pickle
 	@echo
 	@echo "Build finished; now you can process the pickle files."
 
 json: prep
-	$(SPHINXBUILD) -t $(DOC_TAG) -b json $(ALLSPHINXOPTS) $(BUILDDIR)/json
+	$(SPHINXBUILD) -t $(DOC_TAG) -b json source $(BUILDDIR)/json
```

Comment:
If you are going to leave these changes in, can you make it consistent by refering to ${SOURCEDIR} instead of source?